test/dash/dash_parser_segment_list_unit.js

```diff
@@ -371,5 +371,7 @@ describe('DashParser SegmentList', () => {
     ];
     const actual = Array.from(stream.segmentIndex);
     expect(actual).toEqual(expected);
+
+    dashParser.stop();
```

Comment:
If this is outside of afterEach(), and there is an exception thrown earlier, it won't happen.

An alternative to afterEach() is try/finally in the test body:

```js
it('does a thing', () => {
  try {
    // test goes here
  } finally {
    // critical cleanup goes here
  }
});
```

Reply:
Done!